src/el/geth/geth_launcher.star

```diff
@@ -157,8 +157,7 @@ def get_config(
 ):
     # TODO: Remove this once electra fork has path based storage scheme implemented
     if electra_fork_epoch != None:
-        init_datadir_cmd_str = "geth init --cache.preimages --override.prague={0} --datadir={1} {2}".format(
-            final_genesis_timestamp,
+        init_datadir_cmd_str = "geth init --cache.preimages --datadir={0} {1}".format(
```

Comment:
lets wait for the end of the summit!

Reply:
Can we merge this now?

Reply:
@parithosh you sure we can remove the override prague bit?

Reply:
yh cause prague isnt at genesis right, its happening at a defined point in time. So its nothing geth needs to know at genesis, but rather at runtime is enough. You can test it if you like :D